src/common/options.rs

```diff
@@ -145,19 +145,62 @@ impl Endpointing {
     }
 }
 
+/// Endpointing value
+///
+/// See the [Deepgram Endpointing feature docs][docs] for more info.
+///
+/// [docs]: https://developers.deepgram.com/docs/endpointing
+#[derive(Serialize, Deserialize, Debug, PartialEq, Clone, Copy)]
+pub enum Summarize {
+    #[allow(missing_docs)]
+    Enabled(bool),
+
+    #[allow(missing_docs)]
```

Comment:
I almost want to not expose `(true, false)` via this SDK since those old versions are so far in the past, and v2 is the only option we really want people to use.

Maybe just an `Enabled` bool here, and the serialization if true can be "v2"?